src/ngx_http_lua_semaphore.c

```diff
@@ -557,8 +557,10 @@ ngx_http_lua_ffi_sema_gc(ngx_http_lua_sema_t *sem)
         return;
     }
 
-    if (!ngx_queue_empty(&sem->wait_queue)) {
-        ngx_log_error(NGX_LOG_CRIT, ngx_cycle->log, 0,
+    if (!ngx_terminate
+        && !ngx_quit
+        && !ngx_queue_empty(&sem->wait_queue)) {
```

Comment:
This is wrong. `{` should be on a separate line. Be careful. When in doubt, always check the existing code base for the correct coding style.